crates/kornia-image/src/image.rs

```diff
@@ -369,6 +369,109 @@ impl<T, const C: usize, A: ImageAllocator> Image<T, C, A> {
         C
     }
 
+    /// Checks if Image data is stored in contiguous memory
```

Comment:
this comment it's not accurate as the memory is assumed to be contiguous all the time as tensors are mainly constructed by Vec which already fulfill that condition. What this function is checking is another thing